### PR TITLE
testing-wallet

### PR DESCRIPTION
**Pull request by 0xSero**

### CODE REVIEW

File: src/client/components/app/LineDetailsDisplay/LineMetadata.tsx [hunk 1]
```diff
@@ -131,6 +131,8 @@ export const LineMetadata = (props: LineMetadataProps) => {
   const userPositionMetadata = useAppSelector(LinesSelectors.selectUserPositionMetadata);
   const dispatch = useAppDispatch();
 
+  const connectWallet = () => dispatch(WalletActions.walletSelect({ network: 'mainnet' }));
```
Comment (kibagateaux) [inline src/client/components/app/LineDetailsDisplay/LineMetadata.tsx]: getEnv().NETWORK not hardcoded 'mainnet' please

File: src/core/frameworks/blocknative/Onboard.ts [hunk 2]
```diff
@@ -45,14 +45,17 @@ export class BlocknativeWalletImpl implements Wallet {
   }
 
   public create(network: Network, subscriptions: Subscriptions, theme?: Theme): boolean {
-    const networkId = 5;
+    //To do, make network ID Dynamic
```
Comment (kibagateaux) [inline src/core/frameworks/blocknative/Onboard.ts]: isn't this dynamic now? It wasnt before

File: src/client/components/app/LineDetailsDisplay/PositionsTable.tsx [hunk 3]
```diff
@@ -130,9 +137,13 @@ export const PositionsTable = (props: PositionsProps) => {
   //Action Handlers for positions table
 
   const depositHandler = (e: Event) => {
-    //@ts-ignore
-    dispatch(LinesActions.setSelectedLinePosition({ position: e.target.value }));
-    dispatch(ModalsActions.openModal({ modalName: 'addPosition' }));
+    if (!userWallet) {
+      connectWallet();
+    } else {
+      //@ts-ignore
```
Comment (Craigson) [inline src/client/components/app/LineDetailsDisplay/PositionsTable.tsx:143]: What's this //@ts-ignore for?

**Review by Craigson — APPROVED** [review]
Just hte single `@ts-ignore` issue that should be addressed in a new PR.

**Pull request merged**